.env.example

```
@@ -8,9 +8,9 @@ BACKEND_URL=
 # Riven backend api key
 BACKEND_API_KEY=
 # Generate a random secret for better-auth. Use `openssl rand -base64 32` to generate one.
-BETTER_AUTH_SECRET=random_secret
-# Set to the frontend url, also used as ORIGIN
-BETTER_AUTH_URL=http://localhost:5173
+AUTH_SECRET=random_secret
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The AUTH_SECRET key should go before the BACKEND_API_KEY key

(UnorderedKey)